lib/tracker/page_track.rb

```diff
@@ -0,0 +1,19 @@
+require_relative "./controller"
+
+class PageTrack
+  HTTP_ACCEPT = ["text/html"]
```

Comment:
what is the purpose of this?

Reply:
oops i forgot to implement this 😬

Reply:
fixed; https://github.com/WalkerAndCoBrandsInc/tracker/pull/5/files#diff-19293cb5d12611448556bee6818759b0R42